lib/sycamore/sycamore/query/client.py

```diff
@@ -145,13 +146,14 @@ def generate_plan(self, query: str, index: str, schema: OpenSearchSchema) -> Log
         plan = planner.plan(query)
         return plan
 
-    def run_plan(self, plan: LogicalPlan, dry_run=False, codegen_mode=False) -> Tuple[str, str]:
+    def run_plan(
```

Comment:
Why don't we just associate a context with the SycamoreQueryClient?

Reply:
because that would make too much sense :)